protos/sift/campaigns/v1/campaigns.proto

```diff
@@ -10,7 +10,7 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 import "sift/common/type/v1/resource_identifier.proto";
 import "sift/tags/v1/tags.proto";
 
-
+option go_package = "azimuth/gen/protos/go/sift/campaigns/v1;campaignsv1pb";
```

Comment:
Could you remove `go_package` option? We remove this for this repo: https://github.com/sift-stack/sift/blob/8d1e22cd02d965927504a599fb91556319eeb794/makefile#L15